set.mm

```diff
@@ -157581,6 +157522,17 @@ computer programs (as last() or lastChar()), the terminology used for
       VFVGVIVEBQVHCQUFVKUGUHVDVKUIUJUSVCRFZVMVCPUQVARFVBRFVNURABTACTVAVBULUKVCU
       MUNUO $.
 
+    $( The length of a concatenated word.  (Contributed by Stefan O'Rear,
+       15-Aug-2015.)  (Revised by JJ, 1-Jan-2024.) $)
+    ccatlen $p |- ( ( S e. Word A /\ T e. Word B ) ->
+        ( # ` ( S ++ T ) ) = ( ( # ` S ) + ( # ` T ) ) ) $=
+      ( vx cword wcel wa cconcat co chash cfv cc0 caddc cfzo cv wceq fvex lencl
+      cn0 cmin cif cmpt ccatfval fveq2d wfn ifex eqid fnmpti hashfn mp1i syl2an
+      nn0addcl hashfzo0 syl 3eqtrd ) CAFZGZDBFZGZHZCDIJZKLEMCKLZDKLZNJZOJZEPZMV
+      COJGZVGCLZVGVCUAJZDLZUBZUCZKLZVFKLZVEVAVBVMKECDUQUSUDUEVMVFUFVNVOQVAEVFVL
+      VMVHVIVKVGCRVJDRUGVMUHUIVFVMUJUKVAVETGZVOVEQURVCTGVDTGVPUTACSBDSVCVDUMULV
+      EUNUOUP $.
+
```

Comment:
Why is this theorem moved downwards?

Reply:
It accidentally swapped its position with ccatlenOLD.  The OLD version is bound for deletion by its comment. But it cannot be removed, because other OLD theorems still uses it.  All of them will vanish by mid-January, so I did not care much during restoring.